scipy/optimize/tests/test_optimize.py

```diff
@@ -177,17 +178,21 @@ def test_bfgs(self):
 
     def test_bfgs_infinite(self):
         # Test corner case where -Inf is the minimum.  See gh-2019.
-        func = lambda x: -np.e**-x
```

Comment:
Not worth making these changes unless it can be shown that before/after doesnt't/does work. The churn isn't worth it for a change that doesn't look better anyway.

Reply:
`np.e**x` looks so silly to me which is why I changed to `exp`. Indeed the change isn't necessary, so I'll change it back.